docs/en/observability/slo-overview.asciidoc

```diff
@@ -31,17 +31,34 @@ Burn rate::                     The rate at which your service consumes your err
 
 [discrete]
 [[slo-in-elastic]]
-== SLO Dashboard
-From the SLO dashboard list, you can see all of your SLOs and a quick summary of what's happening in each one:
+== SLO overview
+From the SLO overview, you can see all of your SLOs and a quick summary of what's happening in each one:
```

Comment:
Note that I changed this from "SLO Dashboard" to "SLO overview".

1. I don't see any indication in the product that this is called "SLO Dashboard."
2. I wanted to reduce confusion between this content and the new "SLO Dashboard panels." Do you think this change is okay, @mdbirnstiehl ?